database/src/migrations/20210224144754_alter_project_table.ts

```diff
@@ -0,0 +1,68 @@
+import * as fs from 'fs';
+import * as Knex from 'knex';
+import path from 'path';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+/**
+ * Alter the project table:
+ * - add spatial columns
+ * - geom (geometry)
+ * - geog (geography)
+ *
+ * Regenerate the dapi views.
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    set schema '${DB_SCHEMA}';
+    set search_path = ${DB_SCHEMA},public;
```

Comment:
Does this still work if you don't include `public`?

Ideally, we don't reference the public schema anywhere, as its been locked down as much as possible (for security reasons).  But, I know the postgis stuff gets added under public, so not sure if that is a factor here.

Reply:
yup exactly why I had to add it in, to get access to the `geog` type

Reply:
That is one of the compromises we need to make to make that work.